plugins/ProfileExtender/class.profileextender.plugin.php

```diff
@@ -271,7 +271,7 @@ public function SettingsController_ProfileFieldAddEdit_Create($Sender, $Args) {
 
          // Make Options an array
          if ($Options = GetValue('Options', $FormPostValues)) {
-            $Options = explode("\n", preg_replace('/[^\w\s-]/u', '', $Options));
+            $Options = explode("\n", preg_replace('/[^\w\s-()]/u', '', $Options));
```

Comment:
Have you tested this? Usually the dash has to go last or it has a different meaning.